### DISCUSSION TITLE
Since the Node.js version () or package manager  could not be determined, Build Cache will not be stored

### DISCUSSION THREAD

**Discussion by bonesoul** [vercel/community#1103] [Help] [answered]
`Since the Node.js version () or package manager  could not be determined, Build Cache will not be stored`

I'm not sure why I'm getting this on a next.js & node 18 project built over vercel.

**Comment by amyegan** [comment, marked as answer]
Hi @bonesoul! The team released a change last week to address a problem setting the build cache with Node 18. Are you still seeing this warning?
